src/server/main_service.cc

```diff
@@ -1054,10 +1063,14 @@ optional<ErrorReply> Service::VerifyCommandExecution(const CommandId* cid,
   if ((cid->opt_mask() & CO::DENYOOM) && etl.is_master) {
     uint64_t start_ns = absl::GetCurrentTimeNanos();
 
-    uint64_t used_memory = etl.GetUsedMemory(start_ns);
+    auto memory_stats = etl.GetMemoryUsage(start_ns);
     double oom_deny_ratio = GetFlag(FLAGS_oom_deny_ratio);
```

Comment:
Are we ok with reading 2 flags in the hot path?
We could have thread-local caches, and update them upon flag update

Reply:
ok using thread local now